src/orchard/xref.clj

```diff
@@ -47,7 +47,9 @@
   {:added "0.5"}
   [var]
   (let [var (as-var var)
-        all-vars (q/vars {:ns-query {:project? true} :private? true})
+        all-vars (q/vars {:ns-query {:project? true
+                                     :exclude-regexps ["^cider.nrepl.inlined-deps" "^nrepl"]}
```

Comment:
I think we should probably put the list of exclusions in some var that we can just check where this is needed. 

I also wonder if it won't better for the query methods to just filter those out automatically.

Reply:
The `q/namespace` function does filter it out. It just had the wrong prefix. This technically isn't needed.

Reply:
I reset the branch to just the single commit and force pushed. no longer there.